### DISCUSSION TITLE
查看所有图标并可复制classname

### DISCUSSION THREAD

**Discussion by ellisfan** [Tokinx/Wing#147] [讨论]
原来的图标查看地址：https://chuangzaoshi.com/icon/ 无法访问了
我按照css文件重新写了个页面用来查看所有icon，可以点击图标复制classname
https://coze.dev/strawberry-icons

**Comment by Tokinx** [comment]
非常感谢，另外CaoMei Icon的Github地址是：https://github.com/xiangsudian/CaoMei
